logstash-input-tcp.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name          = 'logstash-input-tcp'
-  s.version         = '4.1.2'
+  s.version       = ::File.read('version').split("\n").first
```

Comment:
Moving the version to a separate file can make it difficult for us to automatically version bump plugins as we do during plugin publishing.

What's the story on moving to a version file? (This can be OK, but I just want to see the story)